core/examples/luigi-sample-angular/e2e/tests/navigation.spec.js

```diff
@@ -199,5 +199,43 @@ describe('Navigation', () => {
       cy.get('.no-side-nav').should('exist');
       cy.get('.fd-app__sidebar').should('not.be.visible');
     });
+
+    it('Open navigation node in a modal', () => {
+      let iframeBody;
+      cy.get('iframe', { timeout: 1000 }).then(ifr => {
+        iframeBody = ifr.contents().find('body');
+      });
+      // projects page
+      cy.get('.fd-shellbar')
+        .contains('Projects')
+        .click();
+
+      //projects page
+      cy.get('.fd-app__sidebar').should('contain', 'Project One');
```

Comment:
I would say these two checks for project one and two are not necessary, as well as this one below:
``` 
      cy.get('.fd-app__sidebar').should('not.contain', 'Project Two');
```

Reply:
fine for me :)